defaults/main.yml

```diff
@@ -5,7 +5,7 @@ mysql_hardening_enabled: yes
 mysql_hardening_user: 'mysql'
 mysql_hardening_group: 'root'
 mysql_datadir: '/var/lib/mysql'
-mysql_hardening_hardening_conf: '/etc/mysql/conf.d/hardening.cnf'
+mysql_hardening_hardening_conf: '{{mysql_hardening_mysql_conf_dir}}/hardening.cnf'
```

Comment:
To be consistent, this variable should be named `mysql_hardening_mysql_hardening_conf_file`.